app/controllers/declaration/RoutingCountriesSummaryController.scala

```diff
@@ -52,7 +52,7 @@ class RoutingCountriesSummaryController @Inject()(
     if (countries.nonEmpty) {
       Ok(routingCountriesSummaryPage(mode, RoutingQuestionYesNo.form(), countries))
     } else {
-      navigator.redirectTo(mode, controllers.declaration.routes.RoutingCountriesController.displayRoutingQuestion(_))
+      navigator.continueTo(mode, controllers.declaration.routes.RoutingCountriesController.displayRoutingQuestion(_))
```

Comment:
There seems to be no difference between`continueTo(mode: Mode, factory: Mode => Call)` and `redirectTo(mode: Mode, factory: Mode => Call)`

i wonder if we can get rid of one of them?

Reply:
Ignore - hadn't seen your proposed Navigator changes

Reply:
Yeah, I saw that the methods did similar stuff and we used different method based on the signature. Passing mode is not a big deal, so I tried to use only one method and rest logic make private and inside navigator.

I thought about removing one of those methods, but the PR is big enough so I thought that first cleanup to expose only one method outside of Navigator will be good first step.